src/subdomains/generic/user/models/user/user.entity.ts

```diff
@@ -10,17 +10,22 @@ import { StakingRefReward } from 'src/mix/models/staking-ref-reward/staking-ref-
 import { BankAccount } from 'src/subdomains/supporting/bank/bank-account/bank-account.entity';
 import { CryptoRoute } from 'src/mix/models/crypto-route/crypto-route.entity';
 import { Buy } from 'src/subdomains/core/buy-crypto/route/buy.entity';
+import { Blockchain } from 'src/integration/blockchain/shared/enums/blockchain.enum';
 
 export enum UserStatus {
   NA = 'NA',
   ACTIVE = 'Active',
 }
 
 @Entity()
+@Index('blockchainAddress', (user: User) => [user.address, user.blockchain], { unique: true })
```

Comment:
Remove the index name, is not really helpful
`@Index((user: User) => [user.address, user.blockchain], { unique: true })`